src/xcode/ENA/ENA/Source/Scenes/RecycleBin/Cells/RecycleBinItemCellModel.swift

```diff
@@ -46,7 +50,9 @@ class RecycleBinItemCellModel {
 			case .recovery:
 				return AppStrings.RecycleBin.RecoveryCertificate.headline
 			}
-		case .coronaTest(let coronaTest):
+		case .userCoronaTest:
```

Comment:
```suggestion
		case .userCoronaTest, .familyMemberCoronaTest:
```

Reply:
Implementation wasn't really done yet on the recycle bin (there's a separate subtask), I previously just made it build. But I added the missing texts and images so this makes more sense to have separate cases